test/unit/adb-emu-commands-specs.js

```diff
@@ -256,6 +256,38 @@ describe('adb emulator commands', () => {
         mocks.adb.verify();
       });
     }));
+    describe("network speed method", withMocks({adb}, (mocks) => {
+      it("should throw exception on invalid speed", async () => {
+        await adb.networkSpeed('light').should.eventually.be.rejectedWith("Invalid network speed");
+        mocks.adb.verify();
+      });
+      let speeds = {
+        'gsm': adb.NETWORK_SPEED.GSM,
```

Comment:
But I also don't really understand what this is doing. Doesn't `adb.NETWORK_SPEED.GSM` equal 'gsm'? Would that not mean that `speeds['gsm']` would equal `gsm`? So why not just have the values from `adb.NETWORK_SPEED`?

Here is the `speeds` object:
```js
{ gsm: 'gsm',
  scsd: 'scsd',
  gprs: 'gprs',
  edge: 'edge',
  umts: 'umts',
  hsdpa: 'hsdpa',
  lte: 'lte',
  evdo: 'evdo',
  full: 'full' 
}
```